lib/config.js

```diff
@@ -36,6 +36,7 @@ function config(root) {
  */
 function getDefaultConfig() {
   return {
+    addonName: null,
```

Comment:
You can pass the root from the parent `config` method and do something like this:

`addonName: path.join(root, 'package.json').name`